crates/flox-rust-sdk/src/models/publish.rs

```diff
@@ -968,6 +969,7 @@ mod tests {
 
     #[cfg(feature = "impure-unit-tests")] // /nix/store is not accessible in the sandbox
     #[tokio::test]
+    #[ignore = "disabled as `publish` is deprecated"]
```

Comment:
Should we not just delete the publish tests?

Reply:
Id rather remove the entire publish as a separate commit/PR.
That way we can pull it back in as a whole when we pick up the work on it.